### ISSUE TITLE
duplicate locale file

### ISSUE THREAD

**Issue by jonassmedegaard** [sugarlabs/sugar-toolkit-gtk3#430]
It sems that locale ig and ibo both cover same language - Igbo - one using ISO 639-1 and the other using ISO 639-2/ISO 639-3:

https://en.wikipedia.org/wiki/Igbo_language

Also, it seems wrong for `ig.po` to change the printf identifier in its translation, adding a space from "%d" to "% d".

All in all - without knowing the language(s) involved (and annoyingly the metadata field is not really used), it seems the newer of the files - `ibo.po` should replace the older `ig.po`.

**Comment by quozl** [comment]
I agree, thanks.  `ig.po` from 12a6d2bccba55a75037b9a87013ea0b16c3ae00b seems obviously wrong, given the format directives are changed.  But it does seem functionally harmless;

```log
host:~$ python3 -c 'print("is% d bottles" % 42)'
is 42 bottles
host:~$ python3 -c 'print("is %d bottles" % 42)'
is 42 bottles
host:~$ █
```

`ibo.po` 9b687cab3f1a0da6e4f01330202ed2e8b2b72e9f came from translate.sugarlabs.org.

@samswag, as the Igbo translation team leader, can you explain it?  Please also review https://salsa.debian.org/pkg-sugar-team/sugar-toolkit-gtk3/commit/a2bfc2972cab21c2acbd5c1f3c15a7d14a6228a0 for the change I'm inclined to merge.

**Comment by quozl** [comment]
@chimosky, it seems @samswag is too busy, do you have any skills in this?

**Comment by jonassmedegaard** [comment]
Try make a simple diff between the two po files - there is *zero* 
linguistic differences ;-)

-- 
 * Jonas Smedegaard - idealist & Internet-arkitekt
 * Tlf.: +45 40843136  Website: http://dr.jones.dk/

 [x] quote me freely  [ ] ask before reusing  [ ] keep private

**Comment by samswag** [comment]
How can I help?
